internal/provider/data_source_gitlab_repository_file_test.go

```diff
@@ -59,20 +60,24 @@ func testAccDataSourceGitlabRepositoryFile(src, n string) resource.TestCheckFunc
 }
 
 func testAccDataGitlabRepositoryFile(project string) string {
+	var executeFilemodeConfig string
+	if isExecuteFilemodeSupported, err := isGitLabVersionAtLeast(context.Background(), testGitlabClient, "14.10")(); err == nil && isExecuteFilemodeSupported {
+		executeFilemodeConfig = "execute_filemode = true"
+	}
```

Comment:
@timofurrer  - Out of curiosity, why this method instead of creating a separate test with a skipfunc? This certainly works, but it's not necessarily the pattern we've followed in the past for cross-version tests.

Reply:
Yes, makes sense in this case!